llvm/tools/sycl-post-link/DeviceGlobals.cpp

```diff
@@ -0,0 +1,119 @@
+//===----- DeviceGlobals.cpp - SYCL Device Globals Pass -------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// See comments in the header.
+//===----------------------------------------------------------------------===//
+
+#include "DeviceGlobals.h"
+#include "Support.h"
+
+#include "llvm/ADT/STLExtras.h"
+#include "llvm/ADT/StringRef.h"
+#include "llvm/IR/Module.h"
+
+#include <cassert>
+
+using namespace llvm;
+
+namespace {
+
+constexpr StringRef SYCL_DEVICE_GLOBAL_SIZE_ATTR = "sycl-device-global-size";
+constexpr StringRef SYCL_UNIQUE_ID_ATTR = "sycl-unique-id";
+constexpr StringRef SYCL_DEVICE_IMAGE_SCOPE_ATTR = "device_image_scope";
+
+/// Converts the string into a boolean value. If the string is equal to "false"
+/// we consider its value as /c false, /true otherwise.
+///
+/// @param Value [in] "boolean as string" value.
+///
+/// @returns \c true if the value of \c Value equals to "true", \c false
```

Comment:
This statement contradicts the brief description above and the code

Reply:
Ops, sorry. I've fixed the `returns` description.